src/App/Pages/Vault/BaseCipherViewModel.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Threading.Tasks;
+using Bit.App.Abstractions;
+using Bit.App.Resources;
+using Bit.Core.Abstractions;
+using Bit.Core.Exceptions;
+using Bit.Core.Models.View;
+using Bit.Core.Utilities;
+using Xamarin.CommunityToolkit.ObjectModel;
+
+namespace Bit.App.Pages
+{
+    public abstract class BaseCipherViewModel : BaseViewModel
+    {
+        private CipherView _cipher;
```

Comment:
This should be below the services, just above `AdditionalPropertiesToRaiseOnCipherChanged`